src/components/ingredients/ChosenProducts.tsx

```diff
@@ -21,12 +22,27 @@ const ChosenProducts: FC<Props> = ({
       });
     });
   };
-  console.log(selectedProducts);
+
+  const handleDelete = (product: Product) => {
```

Comment:
Tohle je hodne otazka stylistiky, ale ja bych asi ocekaval, ze tahle komponenta by mela API:

<ChosenProducts products={selectedProducts} onDelete={(product) => ...}  onMarkedAsPreffered={(product) ... } />

Primarne proto, ze bych nastavovani toho stavu nechal na te komponente ktera ten stav vlastni, coz neni takhle. Ale mozna je to v tuhle chvili overengineering. Necham to na tobe.